docker-compose.yml

```diff
@@ -20,63 +26,116 @@ services:
       - MINIO_ACCESS_KEY=miniouserallcsa
       - MINIO_SECRET_KEY=thedefaultminiosecretneedstobechanged
     command: server /data
+    ## depending on network setup
     ports:
       - "9050:9000"
     healthcheck:
-      test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]
+      test: ["CMD", "curl", "-f", "http://s3:9000/minio/health/live"]
       interval: 30s
       timeout: 20s
       retries: 3
+    networks: 
+      ooNet:
+        ipv4_address: 172.16.0.3
 
   server:
     image: openolitor/openolitor-server
     restart: unless-stopped
+    environment:
+      - TZ=UTC
     volumes:
     # server settings
       - ./config/server/openolitor-server.conf:/etc/openolitor-server/application.conf
       - ./config/server/openolitor-server-logback.xml:/etc/openolitor-server/logback.xml
-      - ./log/server/akka.log:/var/log/openolitor-server/akka.log
+      - ./log/server/:/var/log/openolitor-server/
     depends_on:
-    - db
-    - s3
+      - db
+      - s3
     ports:
-    - "9003:9003"
+      - "9003:9003"
     healthcheck:
-      test: ["CMD", "curl", "-f", "http://server:9003/m1/open/projekt"]
+      test: ["CMD", "curl", "-f", "http://server:9003/csa1/open/projekt"]
       interval: 30s
       timeout: 20s
       retries: 3
+    networks: 
+      ooNet:
+        ipv4_address: 172.16.0.4
 
   client-admin:
     image: openolitor/openolitor-client-admin
+    ports:
+      - "9100:80"
     restart: unless-stopped
     volumes:
       - ./config/client/admin/config.js:/usr/share/nginx/html/environments/config.js
+    networks: 
+      ooNet:
+        ipv4_address: 172.16.0.5
 
   client-kundenportal:
     image: openolitor/openolitor-client-kundenportal
+    ports:
+      - "9110:80"
     restart: unless-stopped
     volumes:
       - ./config/client/kundenportal/config.js:/usr/share/nginx/html/environments/config.js
+    networks: 
+      ooNet:
+        ipv4_address: 172.16.0.6
 
   nginx:
     image: nginx:latest
+    restart: unless-stopped
+    environment:
+      - TZ=UTC
     ports:
       - "8080:9000"
+      - "13423:13423"
     volumes:
-    - ./config/nginx/nginx.conf:/etc/nginx/nginx.conf
+      - ./config/nginx/nginx.conf:/etc/nginx/nginx.conf
+      - ./config/nginx/smtp-proxy-htpasswd:/etc/nginx/htpasswd
     depends_on:
       - client-admin
       - client-kundenportal
       - server
+    networks: 
+      ooNet:
+        ipv4_address: 172.16.0.7
 
   pdf-tool:
     image: eugenmayer/jodconverter:rest
+    restart: unless-stopped
     ports:
       - "9040:8080"
+    networks: 
+      ooNet:
+        ipv4_address: 172.16.0.8
+
+  smtp-server:
```

Comment:
I would not activate the smtp-server in local config. This is dangerous!

Reply:
done!